.github/workflows/roxctl.yml

```diff
@@ -0,0 +1,33 @@
+name: Roxctl Image Scan
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  roxctl:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download roxctl
+      run: curl -sSLO https://mirror.openshift.com/pub/rhacs/assets/latest/bin/Linux/roxctl && chmod +x roxctl
+    - name: Image scan
+      env:
+        ROX_API_TOKEN: ${{ secrets.STACKROX_CI_INSTANCE_API_KEY }}
+        ROX_CENTRAL_ADDRESS: ${{ secrets.STACKROX_CI_INSTANCE_CENTRAL_HOST }}
+      run: ./roxctl image scan -e $ROX_CENTRAL_ADDRESS:443 -i quay.io/stackrox-io/main -o table
```

Comment:
whatabout getting roxctl from the CI central so versions match like: https://github.com/stackrox/rox-ci-image/blob/8825725583407f5cc1bb185c0989154a86712a05/.circleci/config.yml#L167